src/main/archetype/it.tests/pom.xml

```diff
@@ -40,6 +40,11 @@
             <artifactId>junit</artifactId>
             <version>4.13.1</version>
         </dependency>
+        <dependency>
+            <groupId>net.sourceforge.htmlunit</groupId>
+            <artifactId>htmlunit</artifactId>
+            <version>2.37.0</version>
```

Comment:
Any reason why not starting with the newest one 2.45?

Reply:
updated, thanks for spotting